MIDI Editor/talagan_OneSmallStep Change note len - 1.lua

```diff
@@ -0,0 +1,9 @@
+-- @noindex
+-- @author Ben 'Talagan' Babut
+-- @license MIT
+-- @description This is part of One Small Step
+
+package.path = debug.getinfo(1,"S").source:match[[^@?(.*[\/])[^\/]-$]] .."?.lua;".. package.path;
+
+local engine_lib = require "talagan_OneSmallStep/talagan_OneSmallStep Engine lib";
+engine_lib.setNoteLen("1");
```

Comment:
A nice trick you can do to avoid having to keep in sync multiple small files like this:

```-- in the main package file
@provides
  Some source file.lua > Install Name 1.lua
  Some source file.lua > Install Name 2.lua
  Some source file.lua > Install Name Foo.lua
  Some source file.lua > Install Name Bar.lua
```

Then have that one source file detect at runtime what it's meant to be:
```lua
local SCRIPT_NAME <const> = select(2, reaper.get_action_context()):match("([^/\\_]+)%.lua$")
local SLOT <const> = tonumber(script_name:match("%- (%d+)"))
```

Reply:
Thanks ! i'll take a little time to correct my PR on this point.

Reply:
Fixed for the various note lengths.